pkg/cmd/projects/create.go

```diff
@@ -0,0 +1,93 @@
+// Creates a depot project.
+package projects
+
+import (
+	"encoding/json"
+	"fmt"
+
+	corev1 "buf.build/gen/go/depot/api/protocolbuffers/go/depot/core/v1"
+	"connectrpc.com/connect"
+	"github.com/depot/cli/pkg/api"
+	"github.com/depot/cli/pkg/helpers"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdCreate() *cobra.Command {
+	var (
+		token         string
+		orgID         string
+		region        string
+		keepGigabytes int64
+	)
+
+	cmd := &cobra.Command{
+		Use:     "create [flags] <project-name>",
+		Aliases: []string{"c"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Create depot project",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
+				return fmt.Errorf("project name is required")
+			}
+			ctx := cmd.Context()
+			projectName := args[0]
+
+			token, err := helpers.ResolveToken(ctx, token)
+			if err != nil {
+				return err
+			}
+
+			if token == "" {
+				return fmt.Errorf("missing API token, please run `depot login`")
+			}
+
+			projectClient := api.NewSDKProjectsClient()
+			req := corev1.CreateProjectRequest{
+				Name:     projectName,
+				RegionId: region,
+				CachePolicy: &corev1.CachePolicy{
+					KeepBytes: keepGigabytes * 1024 * 1024 * 1024,
+				},
+			}
+			if orgID != "" {
+				req.OrganizationId = &orgID
+			}
+
+			res, err := projectClient.CreateProject(ctx, api.WithAuthentication(connect.NewRequest(&req), token))
```

Comment:
So I don't think this currently authenticates the way we think it does. This is the core API service: https://github.com/depot/api/blob/4632b1a9bd80453284b8661d554b54768d8d2221/src/services/depot.core.v1.ProjectService.ts#L36 and only accepts the org token.

I believe to make this functional in the way we want (i.e. an org admin auth'd with a user token can create a project) we would either neither need to add additional token checks to the core projects service. Or we add the create project to the CLI service in the API.

Perhaps @jacobwgillespie has thoughts on which direction we want to go over time.